build: update dependencies

daimoonis opened the pull request.

Commit:
build: update dependencies

Commit:
fix: update package lock

Commit:
fix: pipeline, update nx to latest

Commit:
fix: typescript-eslint parser version and node types version

Flyrell commented:
Honestly, looks legit. Needs to be tested in the real-world environemnt though. My proposition is to try it on AT via npm link, (but on a clear project, so rebuild the images, remove the node_modules, make ci + npm link) if that works, we can probably release it and worst case scenario we will.

daimoonis commented:
> Honestly, looks legit. Needs to be tested in the real-world environemnt though. My proposition is to try it on AT via npm link, (but on a clear project, so rebuild the images, remove the node_modules, make ci + npm link) if that works, we can probably release it and worst case scenario we will.

Well I did that but installed only from local built libs, not with npm link as it messed up my environment and did not let me correctly install all the deps correctly, with an error as resource not found for dependend unreleased dependency. Do you know how to do that correctly as a scenario with npm link?

Flyrell commented:
No idea. Never had that happened. But I think it might be a bad sign xD I'll try it locally, but I'm currently travelling so in the afternoon.

Flyrell committed:
Publish

 - @betsys-eslint/eslint-config-angular-template@3.0.0-alpha.0
 - @betsys-eslint/eslint-config-angular@3.0.0-alpha.0
 - @betsys-eslint/eslint-config-nestjs@3.0.0-alpha.0
 - @betsys-eslint/eslint-config-typescript@3.0.0-alpha.0
 - @betsys-eslint/eslint-plugin-angular-template-spacing@3.0.0-alpha.0
 - @betsys-eslint/install@3.0.0-alpha.0

Commit:
fix: project dependencies installation

- fixed usage of deprecated libraries

Commit:
fix: peer dependencies

Commit:
fix: upgrade lerna by multiple majors

Flyrell committed:
Publish

 - @betsys-eslint/eslint-config-angular-template@3.0.1-alpha.0
 - @betsys-eslint/eslint-config-angular@3.0.1-alpha.0
 - @betsys-eslint/eslint-config-nestjs@3.0.1-alpha.0

daimoonis committed:
chore: prepare peers for next release (#14)

Co-authored-by: svarcm <martin.svarc@betsys.com>

Flyrell committed:
Publish

 - @betsys-eslint/eslint-config-angular-template@3.0.2-alpha.0
 - @betsys-eslint/eslint-config-angular@3.0.2-alpha.0
 - @betsys-eslint/eslint-config-nestjs@3.0.2-alpha.0

Commit:
chore: :art: prepare for release 3.0.0

Flyrell committed:
chore: prepare typescript config for 3.0.0

Flyrell committed:
Publish

 - @betsys-eslint/eslint-config-angular-template@3.0.0
 - @betsys-eslint/eslint-config-angular@3.0.0
 - @betsys-eslint/eslint-config-nestjs@3.0.0
 - @betsys-eslint/eslint-config-typescript@3.0.0
 - @betsys-eslint/eslint-plugin-angular-template-spacing@3.0.0
 - @betsys-eslint/install@3.0.0
